velox/docs/develop/operators.rst

```diff
@@ -619,6 +619,8 @@ If no sorting columns are specified then the order of the results is unspecified
     - Output column names for each window function invocation in windowFunctions list below.
   * - windowFunctions
     - Window function calls with the frame clause. e.g row_number(), first_value(name) between range 10 preceding and current row. The default frame is between range unbounded preceding and current row.
+  * - inputsSorted
+    - If the inputs are sorted, the Window operator will split the window partition and begin processing it as soon as it receives the data. This allows for parallel processing of the window partitions. However, if the inputs are not sorted, the Window operator will wait until it has obtained all the input data before proceeding. In this case, it will sort the data, split the window partition based on the defined criteria, and then process each window partition sequentially.
```

Comment:
Let's clarify that inputs must be clustered on partition keys and sorted on sorting keys in sorting orders.